server/game/cards/07.2-FP/ShelbyHunt.js

```diff
@@ -0,0 +1,43 @@
+const DudeCard = require('../../dudecard.js');
+
+/** @typedef {import('../../AbilityDsl')} AbilityDsl */
+class ShelbyHunt extends DudeCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Shelby Hunt',
+            playType: 'shootout',
+            ifCondition: () => this.opposingPosseContainsDudeWithHigherBullets(),
+            message: context => 
+                this.game.addMessage('{0} uses {1} to increase bullets and become a stud. {1} cannot be chosen or affected by their opponent\'s abilities', 
```

Comment:
Better, but I prefer one sentence if possible.
```suggestion
                this.game.addMessage('{0} uses {1} to give him +1 bullets, make him a stud and he cannot be chosen or affected by their opponent\'s abilities', 
```